forge-gui/res/cardsfolder/upcoming/academy_wall.txt

```diff
@@ -0,0 +1,8 @@
+Name:Academy Wall
+ManaCost:2 U
+Types:Creature Wall
+PT:0/5
+T:Mode$ SpellCast | ValidCard$ Instant,Sorcery | ValidActivatingPlayer$ You | OptionalDecider$ You | Execute$ TrigDraw | TriggerZones$ Battlefield | TriggerDescription$ Whenever you cast an instant or sorcery spell, you may draw a card. If you do, discard a card. This ability triggers only once each turn.
+SVar:TrigDraw:DB$ Draw | Defined$ Self | NumCards$ 1 | SubAbility$ DBDiscard
```

Comment:
It will be cleaner if you write "if you do" with costs for most cases, see _Abomination of Gudul_